Esri/arcgis-maps-sdk-swift-samples
# [New] Get elevation at point on surface

## Description

This PR implements `Get elevation at point on surface` in `Scenes` category.
[URL to README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/2600fbcf4fb79a06587dd40e069998f18f79f7b6/Shared/Samples/Get%20elevation%20at%20point%20on%20surface/README.md)

[Old iOS sample](https://github.com/Esri/arcgis-runtime-samples-ios/tree/main/arcgis-ios-sdk-samples/Scenes/Get%20elevation%20at%20a%20point)

## Linked Issue(s)

- `swift/issues/3582`

## How To Test

Open the sample and tap on a mountain summit. See if the value makes sense. Location is in [northern Nepal](https://en.wikipedia.org/wiki/Roof_of_the_World).

## Screenshots

![screenshot](https://raw.githubusercontent.com/Esri/arcgis-maps-sdk-swift-samples/2600fbcf4fb79a06587dd40e069998f18f79f7b6/Shared/Samples/Get%20elevation%20at%20point%20on%20surface/get-elevation-at-point-on-surface.png)

## To Discuss

See the comments in code.
